main/main.cpp

```diff
@@ -98,36 +100,58 @@ Replxx::completions_t hook_shared(std::string const& context, int index, std::ve
     }
     // std::cerr << "[" << context << "][" << prefix << "]" << context.size() << ":" << index << "label[" << label << "]" <<
     // std::endl;
-    bool label_files  = strcasecmp(label.c_str(), "files") == 0;
-    bool label_output = strcasecmp(label.c_str(), "output") == 0;
-    bool label_path   = strcasecmp(label.c_str(), "path") == 0;
-    bool label_odir   = strcasecmp(label.c_str(), "odir") == 0;
-    if (label_files || label_output || label_path || label_odir) {
-      std::string path = ".";
-      auto        pos2 = full_filename.find_last_of('/');
-      std::string filename;
-      if (pos2 != std::string::npos) {
+
+    std::string path = ".";
+    auto        pos2 = full_filename.find_last_of('/');
+    std::string filename;
+    if (pos2 != std::string::npos) {
         path     = full_filename.substr(0, pos2);
         filename = full_filename.substr(pos2 + 1);
         prefix_add += path + "/";
         prefix = filename;
-      } else {
+    } else {
         filename = full_filename;
-      }
+    }
+
+    bool label_files  = strcasecmp(label.c_str(), "files") == 0;
+    bool label_output = strcasecmp(label.c_str(), "output") == 0;
+    bool label_path   = strcasecmp(label.c_str(), "path") == 0;
+    bool label_odir   = strcasecmp(label.c_str(), "odir") == 0;
+    bool label_name   = strcasecmp(label.c_str(), "name") == 0;
+      	
+    if (label_name)
+    {
+    	std::vector<Lgraph *> lgs;
+    	std::vector<std::string> name_files;
+    	auto *library = Graph_library::instance("lgdb/");
+
+	    library->each_lgraph([&lgs, &name_files, path](Lg_type_id id, std::string_view name){
+	        (void)name;
```

Comment:
You are using name, no need to have the '(void)name". this is to indicate the compile that an argument is not used